specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2018-06-01-preview/apimdeployment.json

```diff
@@ -646,6 +689,141 @@
     }
   },
   "definitions": {
+    "ResourceSkuResults": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceSkuResult"
+          },
+          "description": "The list of skus available for the service."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The uri to fetch the next page of API Management service Skus."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The API Management service SKUs operation response."
+    },
+    "ResourceSkuResult": {
+      "properties": {
+        "resourceType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of resource the SKU applies to."
+        },
+        "sku": {
+          "$ref": "#/definitions/ResourceSku",
+          "readOnly": true,
+          "description": "Specifies API Management SKU."
+        },
+        "capacity": {
+          "$ref": "#/definitions/ResourceSkuCapacity",
+          "readOnly": true,
+          "description": "Specifies the number of API Management units."
+        }
+      },
+      "description": "Describes an available API Management service SKU."
+    },
+    "ResourceSkuCapacity": {
+      "properties": {
+        "minimum": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The minimum capacity."
+        },
+        "maximum": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The maximum capacity that can be set."
+        },
+        "default": {
+          "type": "integer",
+          "readOnly": true,
+          "format": "int32",
+          "description": "The default capacity."
+        },
+        "scaleType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The scale type applicable to the sku.",
+          "enum": [
+            "automatic",
+            "manual",
+            "none"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceSkuCapacityScaleType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Automatic",
+                "description": "Supported scale type automatic."
+              },
+              {
+                "value": "Manual",
+                "description": "Supported scale type manual."
+              },
+              {
+                "value": "None",
+                "description": "Scaling not supported."
+              }
+            ]
+          }
```

Comment:
Please make sure that enum values under "enum" and values under corrosponding "x-ms-enum" has the same case.